modules/nf-core/gffread/tests/main.nf.test

```diff
@@ -25,10 +25,12 @@ nextflow_process {
 
         then {
             assertAll (
-            { assert process.success },
-            { assert snapshot(process.out).match() },
```

Comment:
Quick question; why not just use this assertion if it works? Shouldn't the `process.out.gffread_gff` be included in this assertion?

Reply:
Using process.out (without a specific emit) causes the files to be duplicated in the snapshot, one for the numbered array, one for the named. By specifying which emits to test, we reduce that duplication. Not a biggie, just makes it easier to assess snapshot mismatch errors. 

These tests are configured such that the first produces only a gtf, the seond only the gff. So I'm only testing the snapshot for the one that is produced, leaving the not produced emit for the other assertion.

Reply:
Got it, thank you for the clarification! I definitely understand simplifying the snapshot, the duplication makes if much more difficult.